### PR TITLE
Set env vars for test results

### PR DESCRIPTION
**Pull request by dgusakov**
There is a case when we need to have information about test result in runtime.
For fixtures we can get it from request.
In other non-fixtures methods env looks like nice option to get this information

### CODE REVIEW

File: src/adcm_pytest_plugin/plugin.py [hunk 1]
```diff
@@ -160,4 +161,14 @@ def pytest_runtest_makereport(item, call):
     # set a report attribute for each phase of a call, which can
     # be "setup", "call", "teardown"
 
+    os.environ["rep_" + rep.when + "_passed"] = str(rep.passed)
```
Comment (madlabman) [inline src/adcm_pytest_plugin/plugin.py:164]: Variable name will be customized by executed procces?

Comment (dgusakov) [reply src/adcm_pytest_plugin/plugin.py:164]: By stage. Passible stages: setup, call, teardown

Comment (madlabman) [reply src/adcm_pytest_plugin/plugin.py:164]: So If I run it with xdist?

Comment (dgusakov) [reply src/adcm_pytest_plugin/plugin.py:164]: xdist starts individual processes for each worker. So they do not share env

**Review by madlabman — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**